extensions/interactions/TextInput/TextInput.py

```diff
@@ -71,4 +71,13 @@ class TextInput(base.BaseInteraction):
         },
         'calculation_id': 'Top10AnswerFrequencies',
         'addressed_info_is_supported': True,
+    }, {
+        # Table with answer counts for top N unresolved answers.
```

Comment:
One thing to check -- if we add a new calculation here does it automatically show up in the stats tab for TextInput? /cc @brianrodri

Reply:
@prasanna08 -- just checking this? Does this PR lead to any changes in the state stats modal in the statistics tab UI of the editor page?

If not then it's good to merge. Thanks!